src/main/webapp/WEB-INF/components/section-row.html

```diff
@@ -0,0 +1,46 @@
+<div class="row">
+    <div class="col-md-3 mt-5">
+        <div class="card deal-card h-100">
+            <img class="card-img-top home-deal-img" src="" alt="">
+            <div class="card-body d-flex flex-column">
+            <h5 class="card-title deal-title"></h5>
+            <p class="card-text deal-text"></p>
+            <p class="card-text deal-poster"></p>
+            <a href="#" class="btn btn-primary align-self-end mt-auto float-right">See More</a>
+            </div>
+        </div>
+    </div>
+    <div class="col-md-3 mt-5">
+        <div class="card deal-card h-100">
+            <img class="card-img-top home-deal-img" src="" alt="">
+            <div class="card-body d-flex flex-column">
+            <h5 class="card-title deal-title"></h5>
+            <p class="card-text deal-text"></p>
+            <p class="card-text deal-poster"></p>
+            <a href="#" class="btn btn-primary align-self-end mt-auto float-right">See More</a>
+            </div>
+        </div>
+    </div>
+    <div class="col-md-3 mt-5">
+        <div class="card deal-card h-100">
+            <img class="card-img-top home-deal-img" src="" alt="">
+            <div class="card-body d-flex flex-column">
+            <h5 class="card-title deal-title"></h5>
+            <p class="card-text deal-text"></p>
+            <p class="card-text deal-poster"></p>
+            <a href="#" class="btn btn-primary align-self-end mt-auto float-right">See More</a>
+            </div>
+        </div>
+    </div>
```

Comment:
Seems like each of these blocks are the same? Consider putting in a separate file and just repeating here using include?

Reply:
I deleted the html file instead cos im creating the deal card elements in the home-page.js now